enterprise/server/util/redisutil/redisutil_test.go

```diff
@@ -50,7 +50,7 @@ func genOptions(scheme, user, password, addr, database string) *redis.Options {
 			}
 			addr = net.JoinHostPort(host, port)
 			if scheme == "rediss" {
-				tls_config = &tls.Config{ServerName: host}
+				tls_config = &tls.Config{ServerName: host, MinVersion: tls.VersionTLS12}
```

Comment:
Is this related to the go-redis upgrade? Setting the min version seems fine but if it's not related to the upgrade let's merge that separately?

Reply:
Note that this diff hunk only updates our test data to be consistent with the upstream settings.